docs/source/more_types.rst

```diff
@@ -874,14 +874,16 @@ value of type :py:class:`Coroutine[Any, Any, T] <typing.Coroutine>`, which is a
     :ref:`reveal_type() <reveal-type>` displays the inferred static type of
     an expression.
 
-If you want to use coroutines in Python 3.4, which does not support
-the ``async def`` syntax, you can instead use the :py:func:`@asyncio.coroutine <asyncio.coroutine>`
-decorator to convert a generator into a coroutine.
+If you use coroutines in code that was written for Python 3.4, which
```

Comment:
Should we just drop this? 3.4 is long dead

Reply:
I guess there may still legacy code that was originally written to support Python 3.4 and was never migrated to the new syntax, but this is pretty marginal. At least we could make this section shorter.